labelbox/__init__.py

```diff
@@ -1,6 +1,10 @@
 name = "labelbox"
 __version__ = "3.43.0"
 
+from backports.datetime_fromisoformat import MonkeyPatch
+
+MonkeyPatch.patch_fromisoformat()
```

Comment:
is this change hard to implement so a backport is justified? in the words of their creator would possibly break when a feature becomes present -  see here https://pypi.org/project/backports/ - last paragraph titled **What if the feature is present?**

Reply:
While supporting the reported case is easy enough to fix, there are several edge cases for parsing datetimes and we might miss some of them with a custom implementation. This is best left to a library. The risk of this breaking in a future version is very low since the backport is based on the CPython code itself.